app/services/companyReview.js

```diff
@@ -1,47 +1,35 @@
-const companyService = require('./company');
-const userService = require('./user');
 const companyReviewRepository = require('../repositories/companyReview');
+const companyReviewValidator = require('../validators/companyReview');
 const error = require('./error');
 
 class CompanyReviewService {
   async create(user, companyId, data) {
     const companyReview = {
       ...data,
-      userId: user.id,
       companyId,
+      userId: user.id,
     };
 
-    if (companyReview.id || !companyId || !user) {
-      error.throwValidationError('Invalid company review format.');
-    }
-
-    const companyReviewExists = await companyReviewRepository.getByPropsNonParanoid(companyReview);
-    if (companyReviewExists) {
-      error.throwValidationError('Company review already exists.');
-    }
-
-    await companyService.get(companyId);
+    await companyReviewValidator.validateCreate(companyReview, user, companyId);
```

Comment:
Why didn't you use the same way of using the validator? With `this.validator.validate...`?